lblod/ember-rdfa-editor-lblod-plugins
# GN-4653: configurable, internationalized structure names

### Overview
This PR includes some modifications to the new `structure` node-spec, in order to support configurable and internationalized structure names in the header. Specifically, it includes the following changes:
- Drop `structureName` node-attribute. This attribute has been replaced by both the `structureType` and `displayStructureName` attributes.
- Introduction of a required `structureType` attribute. Examples of `structureType` values are:
  * `article`
  * `title`
  * `chapter`
  * `section`
  * `subsection`
  * `paragraph`
- Introduction of a `displayStructureName` attribute. This attribute controls whether the internationalizated (based on the document language) version of the structure name is displayed inside the header of the structure. The internationalized structure name is based on the `structureType` value and the entries included in the translation files. `displayStructureName` has a default value of `false`.

##### connected issues and PRs:
[GN-4653](https://binnenland.atlassian.net/browse/GN-4653)

### How to test/reproduce
- Start the dummy editor and open the decision sample page
- Insert a decision
- Using the `Structures` menu, insert a decision article
- Notice that, based on the document language, the structure name in the header is correctly internationalized
- In the code, edit the `displayStructureName` attribute value for decision articles. You should notice that the structure name is no longer shown.
- The serialization should also correctly display/hide the structure name based on the `displayStructureName` attribute.

### Challenges/uncertainties
This PR does not yet include support for hiding/showing the structure names of the entries in the table of contents, as we do not yet support the `structure` node in combination with it. Once we do, it should be easy to implement it.

### Checks PR readiness
